mlir-tensorrt/compiler/lib/Dialect/Plan/Transforms/RefineTypes.cpp

```diff
@@ -294,6 +295,30 @@ struct StableHloRefineTypeFromWithShapeGeneric
   }
 };
 
+struct TensorRTResizeAbsorbCastPattern: public OpRewritePattern<tensor::CastOp> {
```

Comment:
I would make this an `OpInterfaceRewritePattern<....>` that operates on the interface type tyou added to the resize ops in the previous MR.

Then make sure to filter on condition ``isa<TensorRTDialect>(definingOp->getDialect())`.